exporters/FusionExporter/palette/dt_weight.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="css/synthesisExporter.css" rel="stylesheet" type="text/css">
+        <script src="js/fusionInterface.js"></script>
+        <script src="js/utility.js"></script>
+    </head>
+    <body>
+        <style>
+            h1{
+                font-size: 18pt;
+            }
+        .dt-weight-title{
+            margin-bottom: 15px;
+        }
+        .dt-weight-input{
+            width: 100%;
+            text-align: center;
+        }
+        #dt-weight-value{
+            width: 100px;
+        }
+        </style>
+        <script src="js/dt_weight.js"></script>
+        <div id="dt-weight">
+            <div class="dt-weight-title">
+                    <h1>Drivetrain Weight</h1>
+                    <p>
+                        This field determines the weight of the drivetrain. Note: This should not include any subsystems.
```

Comment:
Could this be more specific? For example, does this value include the weight of the wheels (since those are also joints)?